src/imitation/scripts/common/common.py

```diff
@@ -45,19 +46,20 @@ def update_log_format_strs(log_format_strs, log_format_strs_additional):
 
 
 @common_ingredient.config_hook
-def hook(config, command_name, logger):
+def hook(config, command_name: str, logger):
     del logger
     updates = {}
     if config["common"]["log_dir"] is None:
         env_sanitized = config["common"]["env_name"].replace("/", "_")
-        log_root = config["common"]["log_root"] or "output"
-        log_dir = os.path.join(
-            log_root,
-            command_name,
-            env_sanitized,
-            util.make_unique_timestamp(),
+        assert isinstance(env_sanitized, str)
+        config_log_root = types.parse_optional_path(config["common"]["log_root"])
```

Comment:
This is quite hard to follow. Can we simplify it? E.g. (perhaps this doesn't work verbatim because of type checking?):
```python
log_root = config["common"]["log_root"] or "output"
log_root = types.parse_path(log_root)
```